src/arviz_plots/plots/ppcplot.py

```diff
@@ -0,0 +1,479 @@
+            plot_collection, aes_map, "predictive", reduce_dims
+        )
+        print(f"\npp_density_aes = {pp_density_aes}\npp_density_ignore= {pp_density_ignore}")
+
+        # getting first default color from color cycle and picking it
+        pp_default_color = plot_bknd.get_default_aes("color", 1, {})[0]
+        if "color" not in pp_density_aes:
+            pp_kwargs.setdefault("color", pp_default_color)
+
+        if "alpha" not in pp_density_aes:
+            pp_kwargs.setdefault("alpha", 0.2)
+
+        plot_dist_artists = ("credible_interval", "point_estimate", "point_estimate_text", "title")
+
+        print(f"\npp_kwargs = {pp_kwargs}")
+
+        # calling plot_dist with plot_collection and customized args
+        pp_dt = DataTree(name="pp_dt", data=pp_distribution)  # has to be converted from a
+        # dataarray to a datatree first before passing to plot_dist
+        plot_dist(
+            pp_dt,
+            var_names=var_names,
+            filter_vars=filter_vars,
+            group="pp_dt",
+            coords=coords,
+            sample_dims=reduce_dims,
+            kind=kind,
+            plot_collection=plot_collection,
+            labeller=labeller,
+            aes_map={
+                kind: value
+                for key, value in aes_map.items()
+                if key == "predictive" and key not in pp_density_ignore
+            },
+            plot_kwargs={
+                **{key: False for key in plot_dist_artists},
+                kind: dict(pp_kwargs.items()),
+            },
+            stats_kwargs={key: value for key, value in stats_kwargs.items() if key == "predictive"},
+        )
+
+    # ---------STEP 2 (PPC AGGREGATE)-------------
+
+    aggregate_kwargs = copy(plot_kwargs.get("aggregate", {}))
+    if aggregate and pp_kwargs is not False and aggregate_kwargs is not False:
+        _, aggregate_density_aes, aggregate_density_ignore = filter_aes(
+            plot_collection, aes_map, "aggregate", reduce_dims
+        )
+
+        print(
+            f"\agg_dens_aes = {aggregate_density_aes}\nagg_dens_ignore= {aggregate_density_ignore}"
+        )
+
+        if "linestyle" not in aggregate_density_aes:
+            aggregate_kwargs.setdefault("linestyle", "--")
+
+        # getting first two default colors from color cycle and picking the second
+        aggregate_default_color = plot_bknd.get_default_aes("color", 2, {})[1]
+        if "color" not in aggregate_density_aes:
+            aggregate_kwargs.setdefault("color", aggregate_default_color)
+
+        print(f"\n aggregate reduce_dims = {reduce_dims!r}")
+        print(f"\n aggregate sample_dims = {sample_dims!r}")
+        print(f"\n aggregate_kwargs = {aggregate_kwargs}")
+        aggregate_reduce_dims = reduce_dims + list(sample_dims)
+        print(f"\n aggregate_reduce_dims = {aggregate_reduce_dims}")
+
+        # setting aggregate aes_map to `[]` so `overlay` isn't applied
+        aes_map.setdefault("aggregate", [])
```

Comment:
This should happen in the aes_map defaults (it doesn't really matter if we set that and then aggregate=False)